examples/Workflows/README.md

```diff
@@ -1,3 +1,5 @@
 # Workflow examples
 
```

Comment:
@maaquib Please add a general guideline here on how the workflow works, what is supported and link to the main doc in serve/doc. Also, a clarification on what should be implemented in the workflow handler (can refer to example for different use-cases), this needs to be updated in the main workflow doc in serve/docs/workflow as well.

Reply:
Done